### ISSUE TITLE
Black pawn colors are wrong

### ISSUE THREAD

**Issue by balbirthomas** [dwcoates/pygn-mode#212]
After installing pygn-mode using `package-install` I ran `pygn-mode-run-diagnostic` and all diagnostics passed (shown below)
```
[x] Good. We can execute the pygn-mode-python-executable at 'python'

[x] Good. The pygn-mode-python-executable at 'python' is a Python 3 interpreter.

[x] Good. The pygn-mode-python-executable at 'python' is better than or equal to Python version 3.7.

[x] Good. The pygn-mode-python-executable at 'python' can import the Python chess library.

[x] Good. The pygn-mode-script-directory ('/home/bt/.emacs.d/elpa/pygn-mode-0.5.1/') is found and the server script is callable.

[x] Good.  The `uci-mode' library is available.

[x] Good.  The `nav-flash' library is available.

[x] Good.  The `ivy' library is available.

------------------------------------

All pygn-mode required diagnostics completed successfully.
```

However on opening a sample PGN file and displaying the board the black pawns are shown in white color (screenshot below).

![screenshot](https://github.com/user-attachments/assets/716c41cb-eca4-4b33-96d2-3c4e8b9e2b4b)

**Comment by rolandwalker** [comment]
Hi!  Something like this happened twice before, and was solved by upgrading the Python `chess` dependency here.  I really don't understand how it breaks or why that solves it, but it sounds like you might understand more.

Anyway let's try updating.

**Comment by rolandwalker** [comment]
Aha!  The update in #214 was supposed to be from  `chess` 1.9.4 to 1.11.1, not so old as 1.6.1.  Are you using https://stable.melpa.org ?  We haven't tagged a release in a long while on the theory that everyone used https://melpa.org instead.  But we can tag a release.

**Comment by rolandwalker** [comment]
Tagged https://github.com/dwcoates/pygn-mode/releases/tag/v0.6.3 .  Not sure how long it will take to show up in MELPA stable.  You were running a _very_ old version.

**Comment by balbirthomas** [comment]
I can confirm this issue is fixed after upgrade to version 0.6.3 on Melap Stable .
